src/main/webapp/i18n/de/exam.json

```diff
@@ -382,8 +382,10 @@
             "startExerciseFailure": "Es gab einen Fehler bei der Vorbereitung der Aufgaben:\n {{message}}",
             "evaluateQuizExerciseSuccess": "{{number}} Quiz-Aufgaben erfolgreich ausgewertet!",
             "evaluateQuizExerciseFailure": "Es gab einen Fehler bei der Auswertung der Quiz-Aufgaben:\n {{message}}",
+            "evaluateQuizExerciseIdFailure": "Quiz-Aufgaben konnten aufgrund einer fehlenden Kurs- oder Prüfung-ID nicht bewertet werden.",
```

Comment:
This error should probably never happen. Imho it should be enough if there is an error in the browser console and the component is not rendered

Reply:
Makes sense, changing this